dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -921,6 +959,176 @@ protected void preValidateLaunchArguments(List<String> args) {
         }
     }
 
+
+    /**
+     * Creates a WES API object and sets the endpoint.
+     *
+     * @param wesUrl URL to WES endpoint
+     */
+    public WorkflowExecutionServiceApi getWorkflowExecutionServiceApi(String wesUrl) {
+        WorkflowExecutionServiceApi clientWorkflowExecutionServiceApi = new WorkflowExecutionServiceApi();
+        //ApiClient wesApiClient = clientWorkflowExecutionServiceApi.getApiClient();
```

Comment:
delete commented code unless there's a reason provided when it might be uncommented

Reply:
Added extra comment on when to uncomment